### PR TITLE
Board auth

### PR DESCRIPTION
**Pull request by mwit-3mdeb**

### CODE REVIEW

File: sortbootorder.c [hunk 1]
```diff
@@ -131,6 +131,8 @@ int main(void) {
 	struct cbfs_handle *bootorder_handle;
 #endif
 
+    lib_get_sysinfo();
```
Comment (miczyg1) [inline sortbootorder.c]: @mwit-3mdeb indentations are not correct

File: sortbootorder.c [hunk 2]
```diff
@@ -143,11 +145,9 @@ int main(void) {
 	noecho(); /* don't echo keystrokes */
 #endif
 
-#ifdef TARGET_APU1
-	printf("\n### PC Engines apu1 setup %s ###\n", SORTBOOTORDER_VER);
-#else
-	printf("\n### PC Engines apu2 setup %s ###\n", SORTBOOTORDER_VER);
-#endif
+    u8 *apu_id_string = lib_sysinfo.mainboard->strings + lib_sysinfo.mainboard->part_number_idx;
```
Comment (miczyg1) [inline sortbootorder.c]: as above

Comment (mwit-3mdeb) [reply sortbootorder.c]: Fixed

File: sortbootorder.c (hunk 1 above)
Comment (mwit-3mdeb) [reply sortbootorder.c]: Fixed

**Pull request merged**